Resources/Prototypes/_FarHorizons/Catalog/VendingMachines/Inventories/mining.yml

```
@@ -10,4 +10,16 @@
   conditions:
   - !type:ListingLimitedStockCondition
     stock: 5
+  - !type:BuyerAccessCondition
+
+- type: listing
+  id: VendorMiningPTK
+  productEntity: PTKFlatpack
+  cost:
+    SalvageTicket: 200
```

Very easy to make 200 tickets last I knew. Since you're also doubling the fire rate of the mining gun this feels more appropriate.... (After some playtests can adjust the price back down if it is too high)
```suggestion
    SalvageTicket: 400
```